manim/mobject/number_line.py

```diff
@@ -68,6 +68,29 @@ class NumberLine(Line):
     kwargs : Any
         Additional arguments to be passed to :class:`~.Line`.
 
+    Examples
+    --------
+    .. manim:: NumLineExample
+        :save_last_frame:
+        class NumLineExample(Scene):
+            def construct(self):
+                l0 = NumberLine(x_range=[-10, 10, 2], length=10, color=BLUE, include_numbers=True, label_direction=UP)
+                l0.shift(2 * UP)
+
+                l1 = NumberLine(x_range=[-10, 10, 2], unit_size=0.5, numbers_with_elongated_ticks=[-2, 4], include_numbers=True, number_scale_value=0.5)
+                l1.shift(UP)
+                num6 = l1.get_number_mobject(6).set_color(RED)
```

Comment:
Since `num6` is associated with `l1`, it should probably be added to it (this also makes the VGroup usage possible)
```suggestion
                num6 = l1.get_number_mobject(6).set_color(RED)
                l1.add(num6)
```

Reply:
Suggested changes are included in the latest commit.